$NetBSD: patch-Makefile,v 1.4 2021/06/10 16:45:22 nia Exp $

Do not pick pkgsrc libtool on Darwin.
Use correct installation dirs.
Avoid non-portable sed in updateversion.

--- Makefile.orig	2020-11-27 18:25:02.000000000 +0000
+++ Makefile
@@ -15,7 +15,7 @@ VERSION = $(shell cat VERSION)
 
 ifneq (, $(findstring darwin, $(OS)))
 JARSUFFIX=mac
-LIBTOOL ?= $(shell which libtool)
+LIBTOOL ?= /usr/bin/libtool
 endif
 ifneq (, $(findstring linux, $(OS)))
 JARSUFFIX=linux
@@ -155,7 +155,7 @@ endif
 INSTALLDIRINC = $(INSTALLPREFIX)/include/miniupnpc
 INSTALLDIRLIB = $(INSTALLPREFIX)/$(LIBDIR)
 INSTALLDIRBIN = $(INSTALLPREFIX)/bin
-INSTALLDIRMAN = $(INSTALLPREFIX)/share/man
+INSTALLDIRMAN = $(INSTALLPREFIX)/$(PKGMANDIR)
 PKGCONFIGDIR = $(INSTALLDIRLIB)/pkgconfig
 
 FILESTOINSTALL = $(LIBRARY) $(EXECUTABLES)
@@ -242,7 +242,7 @@ updateversion:	miniupnpc.h
 	cp miniupnpc.h miniupnpc.h.bak
 	sed 's/\(.*MINIUPNPC_API_VERSION\s\+\)[0-9]\+/\1$(APIVERSION)/' < miniupnpc.h.bak > miniupnpc.h
 
-install:	updateversion $(FILESTOINSTALL)
+install:	$(FILESTOINSTALL)
 	$(INSTALL) -d $(DESTDIR)$(INSTALLDIRINC)
 	$(INSTALL) -m 644 $(HEADERS) $(DESTDIR)$(INSTALLDIRINC)
 	$(INSTALL) -d $(DESTDIR)$(INSTALLDIRLIB)
